### DISCUSSION TITLE
Error running query: 'Result' object has no attribute 'json_data'

### DISCUSSION THREAD

**Discussion by jessica-bauer** [getredash/redash#6052] [Q&A] [answered]
I get the error "Error running query: 'Result' object has no attribute 'json_data'" when I run the following query:

![image](https://github.com/getredash/redash/assets/79603087/bd54b039-bd57-4938-bbcb-fb9b39cd66e2)

```
SELECT
    GROUP_CONCAT(DISTINCT s.first_name SEPARATOR '\n\n')  AS 'First Name', 
    GROUP_CONCAT(DISTINCT s.last_name SEPARATOR '\n\n')  AS 'Last Name',
    s.id  AS 'ID',
    GROUP_CONCAT(DISTINCT CONCAT_WS(' - ', oi.order_id, oi.description, tech.`Product Name`) ORDER BY oi.order_id SEPARATOR '\n\n')  AS 'Order ID - Item Description',
    GROUP_CONCAT(DISTINCT inv.asset ORDER BY oi.description SEPARATOR '\n\n')  AS 'Asset',
    GROUP_CONCAT(DISTINCT oi.item_id ORDER BY oi.order_id SEPARATOR '\n\n') AS 'Item ID',
    GROUP_CONCAT(DISTINCT o_r.order_type ORDER BY oi.order_id SEPARATOR '\n\n')  AS 'Order Type',
    GROUP_CONCAT(DISTINCT inv.status SEPARATOR '\n\n')  AS 'Status',
    GROUP_CONCAT(DISTINCT inv.condition SEPARATOR '\n\n') AS 'Item Condition'
FROM                
    school_one.inventory inv
    LEFT JOIN school_one.students s ON inv.student_id = s.student_id
    LEFT JOIN school_one.order_items oi ON inv.item_id = oi.order_item_id
    LEFT JOIN school_one.order_requests o_r  ON oi.order_request_id  = o_r.order_request_id
    LEFT JOIN db.all_students_inventory_school_one tech ON oi.order_id  = tech.`Order ID`
WHERE
    inv.status = 'Processed'
GROUP BY
    s.id;
```

I have 2 other queries for `school_two` and `school_three` schemas that are similar and execute without the error. I was able to isolate the issue with the `db.all_students_inventory_school_one` view, which gives me the same error when I execute it alone. The view is also similar for the other 2 school schemas (`db.all_students_inventory_school_two` and `db.all_students_inventory_school_three`) so I'm not sure if the error might have to do with the data I'm querying.

Thanks for any help. If further info or clarification is needed, please let me know.

**Comment by jessica-bauer** [comment, marked as answer]
Issue has been resolved. There was a statement in the WHERE clause in a joined table that was creating the error. Removing the statement resolved the error issue.

**Comment by bigwheel** [comment]
In our case, it was due to two same column names in select.
This error might be caused by upgrade v8 → v10.

**Reply by jessica-bauer** [reply]
Thanks for that. Will be good to know in any future cases. 

I don't have any columns with the same name and we're currently using v10. I still occasionally getting the error when the query refreshes (I have it on a daily refresh schedule) and then some days I don't get the error. I'm still puzzled by it, but it will eventually be archived and no longer be used.
